testing-guidance/workflow-tests/ragdoll/ragdoll-authoring-improvements.md

```diff
@@ -0,0 +1,47 @@
+| **Self-collision Warning** | <ol><li>Make sure that the Ragdoll Collider drawing is enabled under the "eye" button dropdown menu.</li><li>See if there are any bones present with red wiremeshes. On the skeleton set up in these workflows, these should be "L_leg_INT", and "R_leg_INT". Red coloration of the wiremesh means that two joints **which are not adjecent in the hierarchy** (for example "R_knee_INT" bone is adjacent to "R_leg_INT", and "R_foot_INT") are colliding with each other. This can be fixed by removing the collision of one of the intersecting bones (select a bone in the Skeleton Outliner, and choose Ragdoll → Remove colliders), or by simply moving/rotating the bones so that they do not intersect, by changing their Offset/Rotation properties. </li> <li>Select a bone, and move/rotate it around by modifying the collider's Offset/Rotation properties under the Ragdoll pane, or using relevant manipulators in the Atom Render Window. Note the change of the wiremeshes color from grey to red when intersecting with other bones that are not adjacent in the bone hierarchy.</li></ol> | <ul><li>Ragdoll Collider drawing can be enabled, and draws wiremeshes of the bones added to the ragdoll.</li><li>Intersecting non-adjacent in hierarchy bones have red wiremeshes instead of grey.</li><li>Moving/rotating bones around properly updates the wiremesh coloration if an unallowed intersection occurs.</li></ul>  |
+| **Picking and hovering** | <ol><li>Select different joints by clicking on their wiremeshes in the Atom Render Window.</li><li>Hover over different bones in the Skeleton Outliner, if their wiremesh is present in the Atom Render Window, they should highlight in a bright color.</li></ol> | <ul><li>Bones can be selected by clicking their wiremeshes in the Atom Render Window.</li><li>Hovering over bones in the Skeleton Outliner highlights them in the Atom Render Window.</li><li>If hovering over different bones does not work, please enable/disable the "Line Skeleton" under the "eye" button dropdown in the Atom Render Window.</li></ul>  |
```

Comment:
Step 2: Move the validation of wiremesh hover highlight behavior to the Things to Watch Out For.

Reply:
Adjusted step 2 as requested.